openedx/core/djangoapps/discussions/config/waffle.py

```diff
@@ -0,0 +1,42 @@
+"""
+This module contains various configuration settings via
+waffle switches for the discussions app.
+"""
+
+from edx_toggles.toggles import LegacyWaffleFlagNamespace
+
+from openedx.core.djangoapps.waffle_utils import CourseWaffleFlag
+
+
+WAFFLE_NAMESPACE = LegacyWaffleFlagNamespace(name='discussions')
+
+# .. toggle_name: discussions.override_discussion_legacy_settings
+# .. toggle_implementation: CourseWaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Waffle flag to override visibility of discussion settings for legacy experience.
+# .. toggle_use_cases: temporary, open_edx
+# .. toggle_creation_date: 2021-06-15
+# .. toggle_target_removal_date: 2021-12-31
+# .. toggle_warnings: Discussion settings will be visible when this flag is enabled with Pages & Resources flag enabled.
```

Comment:
This should warn the user what happens when this flag is turned on. 
so ideally, this should state, When the discussion MFE is enabled, turning this flag ON will port the course back to the legacy view. or anything similar.